spec/acceptance/init_spec.rb

```diff
@@ -144,4 +144,20 @@ class { helm_binary:
       it { is_expected.to be_linked_to '/opt/helm/3.5.4/bin/helm' }
     end
   end
+
+  context 'with packaging set to package' do
+    let(:manifest) do
+      <<-PP
+      class { helm_binary:
+        packageing => 'package',
```

Comment:
I suspect `packageing` is the source of the CI failure.

Reply:
it says the class has no such parameter, but it has, its part of the patch 🤔

Reply:
ah damn, its a typo m(